src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const FORMAT_SOURCE = sourceString.split('\n').join('');
```

Comment:
[CODE KNOWLEDGE] - Don't need to make copy before each operation. Moreover, methods like `filter` or `map` don't mutate the original array. In this line, the split and join operations are used to remove newlines, which could be done directly in line 14 where you map over the styles. You could use `sourceString.trim().split(/\s*;\s*/)` to split the string into declarations, handling optional whitespace around semicolons.